code/game/gamemodes/malfunction/newmalf_ability_trees/HELPERS.dm

```diff
@@ -182,7 +182,7 @@
 /proc/get_hacked_apcs(var/mob/living/silicon/ai/user)
 	var/list/H = list()
 	for(var/obj/machinery/power/apc/A in machines)
-		if(!A.hacker && !A.hacker == user)
+		if(!A.hacker || !A.hacker == user)
```

Comment:
`!A.hacker == user` is still invalid.

`A.hacker == user` or `A.hacker != user`